content/shell/renderer/shell_content_renderer_client.cc

```diff
@@ -277,6 +283,69 @@ void ShellContentRendererClient::GetSupportedKeySystems(
 }
 #endif
 
+#if BUILDFLAG(IS_COBALT)
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+// TODO(b/376542844): Eliminate the usage of hardcoded MIME string once we
+// support to query codec capabilities with configs.
+std::string GetMimeFromAudioType(const media::AudioType& type) {
+  // The MIME string is for very basic audio codec supportability check.
+  switch (type.codec) {
+    case media::AudioCodec::kAAC:
+      return "audio/mp4; codecs=\"mp4a.40.2\"";
+    case media::AudioCodec::kAC3:
+      return "audio/mp4; codecs=\"ac-3\"";
+    case media::AudioCodec::kEAC3:
+      return "audio/mp4; codecs=\"ec-3\"";
+    case media::AudioCodec::kOpus:
+      return "audio/webm; codecs=\"opus\"";
+    // TODO(b/375232937): Support IAMF
+    default:
+      return "";
+  }
+}
+
+std::string GetMimeFromVideoType(const media::VideoType& type) {
+  // The MIME string is for very basic video codec supportability check.
+  switch (type.codec) {
+    case media::VideoCodec::kH264:
+      return "video/mp4; codecs=\"avc1.4d4015\"";
+    case media::VideoCodec::kVP9:
+      return "video/webm; codecs=\"vp9\"";
```

Comment:
`type` has information on profile.  I am not sure if it's parsed and used in the caller.  If it does, then the profile information gets lost here.  This can sometimes cause issues.  For example, vp9 profile 2 indicates hdr support, so an implementation accepts `codecs=vp9` may reject `codecs=vp9.2`.

Feel free to add this as a TODO.

Reply:
The TODO was added at line 288. Added it for this function too.

Reply:
It'll be great to add specific use case (i.e. capture the above comment) to the comment.